libs/platform/src/lib/components/info-label/info-label.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, ViewEncapsulation, ChangeDetectionStrategy, Input, ElementRef } from '@angular/core';
+
+type LabelType = 'numeric' | 'only-icon' | 'icon';
+
+@Component({
+    selector: 'fdp-info-label',
+    templateUrl: './info-label.component.html',
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class InfoLabelComponent {
+
+    /**
+     * The LabelType represented by the info label .
+     * Can be one of the following: 'numeric' | 'only-icon' | 'icon'
+     * For default info label omit this property
+     */
+    @Input()
+    type: LabelType;
```

Comment:
Is this completely new PR. I was under assumptions that I have already commented this file but now I dont see my comments

Reply:
Wrong button, I wanted to select request change. As we cannot close old PR with comments and create new one, just because we have some merge issues. This way people need to start again with review.